src/status_im/contexts/wallet/create_account/edit_derivation_path/view.cljs

```diff
@@ -38,83 +73,106 @@
                              (when on-reset
                                (on-reset)))]
     (fn [{:keys [theme]}]
-      [rn/view
-       {:style (style/screen top)}
-       [quo/page-nav
-        {:background :blur
-         :icon-name  :i/close
-         :on-press   #(rf/dispatch [:navigate-back])}]
-       [quo/text
-        {:size   :heading-1
-         :weight :semi-bold
-         :style  style/header}
-        (i18n/label :t/edit-derivation-path)]
-       [rn/view {:style style/tag}
-        [quo/context-tag
-         {:type    :icon
-          :size    24
-          :icon    :i/placeholder
-          :style   style/tag
-          :context "Alisher Card"}]]
-       [rn/view {:style style/temporal-placeholder}
-        [quo/text "Dropdown input will be here"]
-        [quo/text (i18n/label :t/path-format)]]
-       [quo/input
-        {:container-style style/input-container
-         :value           @path-value
-         :label           (i18n/label :t/derivation-path)
-         :placeholder     (utils/get-formatted-derivation-path 3)
-         :button          {:on-press reset-path-value
-                           :text     (i18n/label :t/reset)}
-         :on-change-text  handle-path-change}]
+      (let [{:keys [public-key]}   (rf/sub [:profile/profile])
+            primary-name           (first (rf/sub [:contacts/contact-two-names-by-identity
+                                                   public-key]))
+            profile-picture        (rf/sub [:profile/image])
+            show-path-format-sheet #(rf/dispatch [:show-bottom-sheet {:content path-format-sheet}])]
+        [rn/view
+         {:style (style/screen top)}
+         [quo/page-nav
+          {:background :blur
+           :icon-name  :i/close
+           :on-press   #(rf/dispatch [:navigate-back])}]
+         [rn/view {:style {:padding-bottom 20}}
+          [quo/text
+           {:size   :heading-1
+            :weight :semi-bold
+            :style  style/header}
+           (i18n/label :t/edit-derivation-path)]
+          [rn/view {:style style/tag}
+           [quo/context-tag
```

Comment:
can be:
```
           [quo/context-tag
              {:container-style style/tag
```

Reply:
and remove the wrapping view

Reply:
@J-Son89 the wrapping view seems to be needed, otherwise the context-tag component fills the whole space

Reply:
thanks for checking!